messaging-activemq/injection/src/test/java/org/wildfly/extension/messaging/activemq/deployment/injection/AbstractJMSContextTestCase.java

```diff
@@ -42,7 +42,7 @@
 /**
  * Test class for AbstractJMSContextTes.
  *
- * @author <a href="http://fnbrandao.com.br/">Fabio Nascimento Brand�o</a> (c) 2021 Red Hat inc.
+ * @author <a href="http://fnbrandao.com.br/">Fabio Nascimento Brandão</a> (c) 2021 Red Hat inc.
```

Comment:
Has already been fixed: https://github.com/wildfly/wildfly/commit/2441740e1621a99a7f9517ab6d1e6aac306d64e3

Reply:
@boris-unckel no problem, since the fix is the same, git will just skip this.